DQMOffline/Muon/python/muonEfficiencyAnalyzer_cfi.py

```diff
@@ -170,11 +171,61 @@
                                                       phiMax = cms.double(3.2),
                                                       phiMin = cms.double(-3.2),
                                                       
-                                                      vtxBin = cms.int32(10),
-                                                      vtxMax = cms.double(40.5),
-                                                      vtxMin = cms.double(0.5),
-                                                      
+                                                      vtxBin = cms.int32(30),
+                                                      vtxMax = cms.double(149.5),
+                                                      vtxMin = cms.double(0.5),                                                      
                                                       ID = cms.string("Medium"),
                                                       folder = cms.string("Muons_miniAOD/EfficiencyAnalyzer/")
-                                                      
                                                       )
+
+
+EfficiencyAnalyzer = cms.Sequence(TightMuonEfficiencyAnalyzer*LooseMuonEfficiencyAnalyzer*MediumMuonEfficiencyAnalyzer)
+
+EfficiencyAnalyzer_miniAOD = cms.Sequence(TightMuonEfficiencyAnalyzer_miniAOD*LooseMuonEfficiencyAnalyzer_miniAOD*MediumMuonEfficiencyAnalyzer_miniAOD)
+
+TightMuonEfficiencyAnalyzer_Phase2=TightMuonEfficiencyAnalyzer.clone()
+TightMuonEfficiencyAnalyzer_Phase2.vtxBin=20
+TightMuonEfficiencyAnalyzer_Phase2.vtxMax=249.5
+TightMuonEfficiencyAnalyzer_Phase2.vtxMin=149.5
```

Comment:
```suggestion
TightMuonEfficiencyAnalyzer_Phase2=TightMuonEfficiencyAnalyzer.clone(
  vtxBin=20
  vtxMax=249.5
  vtxMin=149.5
  )
```
and the same here below